### PR TITLE
Resolving several issues

### PR DESCRIPTION
**Pull request by tigerpaws01**
Updates:
- Removed silence detector. To stop recording, either cancel it or start transcribing.
- Forced screen to be always on when recording.
- Added a space button
- Added an option to opt out automatic recording start when keyboard is first shown.
- Added dark mode support (requires >= API Level 29).
- Adjusted keyboard layout.

Closes: #26 
Closes: #27
Closes: #29 
Closes: #30 
Closes: #31

### CODE REVIEW

File: android/app/src/main/java/com/example/whispertoinput/WhisperInputService.kt [hunk 1]
```diff
@@ -184,10 +190,17 @@ class WhisperInputService : InputMethodService() {
         recorderManager!!.stop()
 
         // If this is the first time calling onWindowShown, it means this IME is just being switched to
-        // Automatically starts recording after switching to Whisper Input
-        if (isFirstTime) {
+        // Automatically starts recording after switching to Whisper Input (if settings enabled)
+        // Dispatch a coroutine to do this task.
+        CoroutineScope(Dispatchers.Main).launch {
+            if (!isFirstTime) return@launch
```
Comment (j3soon) [inline android/app/src/main/java/com/example/whispertoinput/WhisperInputService.kt:196]: Self note:

https://kotlinlang.org/docs/returns.html#return-to-labels

**Review by j3soon — COMMENTED** [review]
The code looks good to me, and I've left some comments. Thank you!

File: android/app/src/main/java/com/example/whispertoinput/settings/SettingsPageBuilder.kt
[the remark this answers is not in the document]
Comment (tigerpaws01) [reply android/app/src/main/java/com/example/whispertoinput/settings/SettingsPageBuilder.kt:1]: A comment is added to the commit.

**Review by j3soon — APPROVED** [review]

**Comment by luisherranz** [comment]
Am I the only one who has tried to install [version 0.2](https://github.com/j3soon/whisper-to-input/releases/tag/v0.2) and encountered an error?

image

**Comment by j3soon** [comment]
@luisherranz This is strange... Could you open an issue and provide more details such as the hardware and software info of your phone?

**Comment by luisherranz** [comment]
Never mind, don't worry, I fixed it following [these steps](https://www.reddit.com/r/androidapps/comments/u9drlb/i_want_to_install_an_apk_but_it_shows_app_not/).

**Pull request merged**